terraform/aws/variables.tf

```diff
@@ -56,6 +56,11 @@ variable "tendermint_p2p_port" {
   default     = 36656
 }
 
+variable "prometheus_port" {
+  description = "Prometheus port"
+  default     = 80
```

Comment:
Is it actually prometheus default port?

Reply:
Default is 9090, we set 80 so it's easier to access. Let's not change this

Our default and Prometheus default are different